### DISCUSSION TITLE
External powered hub for the Coral USB Accelerator ( is it necessary )

### DISCUSSION THREAD

**Discussion by stamandr** [blakeblackshear/frigate#4211] [Ask A Question]
I read somewhere that an RaspPI needs an external powered hub for the Coral USB Accelerator. Is this also recommended and necessary for an Intel NUC or Tiny PC ? 

Also, do you need to add an USB extension cable to the Coral USB Accelerator ?

**Comment by NickM-27** [comment]
Extension cords are not necessary as the Coral does not send or receive radio signals. 

As far as external powered hub, depends on how much power your motherboard / USB ports are capable of putting out.

**Reply by stamandr** [reply]
I have no idea on this. I have a Intel NUC 6 (NUC6i5SYK)
https://ark.intel.com/content/www/us/en/ark/products/89188/intel-nuc-kit-nuc6i5syk.html

and HP G2 Mini
https://support.hp.com/ca-en/product/hp-elitedesk-800-65w-g2-desktop-mini-pc/8741496/product-info
